lib/msal-core/src/authority/AuthorityFactory.ts

```diff
@@ -9,25 +9,42 @@
 import { AadAuthority } from "./AadAuthority";
 import { B2cAuthority } from "./B2cAuthority";
```

Comment:
Other end of the circle

Reply:
Per @tnorling this circular dep was resolved in https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/1479